programs/clearing_house/src/math/amm_jit.rs

```diff
@@ -10,26 +11,98 @@ use solana_program::msg;
 pub fn calculate_jit_base_asset_amount(
     market: &PerpMarket,
     taker_base_asset_amount: u64,
```

Comment:
to be clear, the max jit amount should never be more than the "maker" intended base asset amount

Reply:
nice